### DISCUSSION TITLE
Copy-DbaSsisCatalog warning "this command is not supported on linxu or macos"

### DISCUSSION THREAD

**Discussion by murali-cygenex** [dataplat/dbatools#9085] [Q&A]
Dear Dbatools users,

I am having this weird error which says Copy-DbaSsisCatalog warning "this command is not supported on linxu or macos", however I am running dbatools on Windows server 2019.

I have checked the powershell script, the code says " PSVersionTable.PSEdition -eq "Core" Stop-Function -Message "This command is not supported on Linux or macOS". It doesn't make sense, above powershell 4.1 PSEdition value always come "Core" and execution will never go beyond this point.

any help is highly appreciated.

thanks
Murali

**Comment by potatoqualitee** [comment]
That is super weird. It should be reporting Desktop or Nothing. Are you using powershell.exe or the ISE?
